rust/ql/lib/codeql/rust/dataflow/internal/DataFlowImpl.qll

```diff
@@ -160,11 +165,6 @@ module Node {
      * Gets the control flow node that corresponds to this data flow node.
      */
     CfgNode getCfgNode() { none() }
-
-    /**
-     * Gets this node's underlying SSA definition, if any.
-     */
-    Ssa::Definition asDefinition() { none() }
```

Comment:
This method doesn't seem to have a non-`none` definition anywhere and is not used.  Was it just copied from another language or is there more going on here?

Reply:
It was probably just copied.